dev/ants.jl

```diff
@@ -0,0 +1,14 @@
+using Pkg
```

Comment:
What's the `dev` dir for? Just transient while you work on this?

Reply:
Sorry, yes. Just while it's a draft. None of `dev` will stay